docs/index.rst

```diff
@@ -103,6 +105,9 @@ Compare & Contrast with Alternatives
 
 There are several alternatives that create isolated environments:
 
+* Python 3's `venv module <https://docs.python.org/3/library/venv.html>`_
```

Comment:
I would like to reformulate this a bit. Virtualenv is still the de-facto tool to use if you want cross python virtual environment creation; custom script activations (powershell, xonosh) or support for pypy, jython and co. It also has a few features desire-able in some situations.